src/styles/plot.scss

```diff
@@ -1,36 +1,30 @@
 $rv-xy-plot-axis-font-color: #6B6B76;
-$rv-xy-plot-axis-line-color: #e6e6e9;
+$rv-xy-plot-axis-line-color: #E6E6E9;
 $rv-xy-plot-axis-font-size: 11px;
-
 $rv-xy-plot-tooltip-background: #3A3A48;
-$rv-xy-plot-tooltip-color: #fff;
+$rv-xy-plot-tooltip-color: #FFF;
 $rv-xy-plot-tooltip-font-size: 12px;
 $rv-xy-plot-tooltip-border-radius: 4px;
 $rv-xy-plot-tooltip-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
```

Comment:
while were linting stuff, prefer rbga with hexes: rgba(#000, 0.5) would be better here